bot.py

```diff
@@ -28,8 +30,14 @@ async def on_command_error(ctx, error):
 		return
 	await ctx.send("An error occured!\n```{}```".format(error))
 '''
-		
-for cog in os.listdir(".\\commands"):#path
+
+curdir = os.getcwd()
+curdir.split('\\')
+if "Synergile" not in curdir:
+	os.chdir("Synergile")
+else:
+	pass
+for cog in os.listdir(f"{os.getcwd()}{os.path.sep}commands"):#path
```

Comment:
I'd like to get this defined as a calculated absolute path if possible. There should be some way of getting the directory of the base script programmatically, regardless of what the cwd is at the current moment.